packages/validator/src/validator.ts

```diff
@@ -86,6 +86,9 @@ export class Validator {
   /** Waits for genesis and genesis time */
   static async initializeFromBeaconNode(opts: ValidatorOptions, signal?: AbortSignal): Promise<Validator> {
     const {config} = opts.dbOps;
+    // TODO [DA] is there any objection to renaming api to client?
+    // it then becomes obvious this is a client that makes a request to an api
+    // instead of this being an api that is being made available
```

Comment:
Sounds good to me! `api` name is an old legacy option so feel free to change

Reply:
Good to know! I'll remove the comment now but do the renaming in a separate PR (with other renaming propositions) as I think it is different from the main changes in this PR. I left that comment as just questions I was having about the code not necessary intended to be part of this change.